x-pack/test/reporting_api_integration/reporting_and_security/usage/api_counters.ts

```diff
@@ -92,39 +92,53 @@ export default function ({ getService }: FtrProviderContext) {
       });
 
       it('downloading', async () => {
-        await supertestUnauth
-          .get('/api/reporting/jobs/download/kraz0qle154g0763b569zz83')
-          .auth('test_user', 'changeme');
-        await supertestUnauth
-          .get('/api/reporting/jobs/download/kraz0vj4154g0763b5curq51')
-          .auth('test_user', 'changeme');
-        await supertestUnauth
-          .get('/api/reporting/jobs/download/k9a9rq1i0gpe1457b17s7yc6')
-          .auth('test_user', 'changeme');
+        try {
+          await Promise.all([
```

Comment:
I'm not entirely sure why the test was consistently failing before. I expected that I would have to download _fewer_ report jobs to avoid the network errors that were failing the tests. However, changing to `Promise.all` in this part of the code seemed to solve it.